Package.swift

```diff
@@ -158,33 +162,36 @@ let package = Package(
         ),
         .executableTarget(
             name: "plutil",
-            dependencies: ["Foundation"]
+            dependencies: [
+                .targetItem(name: "Foundation", condition: nil)
+            ]
         ),
         .executableTarget(
             name: "xdgTestHelper",
             dependencies: [
-                "Foundation",
-                "FoundationNetworking"
+                .targetItem(name: "Foundation", condition: nil),
+                .targetItem(name: "FoundationXML", condition: nil),
```

Comment:
I think this dependency is new. I don't expect it to be a problem, but was it intentional?

Reply:
Yup, it was intentional.